### DISCUSSION TITLE
Allure json specification?

### DISCUSSION THREAD

**Discussion by wingyplus** [allure-framework/allure2#1493] [Questions & Support] [answered]
Hi, 

I've planned to experiment Allure reporter for Elixir programming language but I didn't see any reference for JSON file specification. Do we have any documentation about it?

**Comment by baev** [comment, marked as answer]
The JSON format is described in the docs https://allurereport.org/docs/how-it-works-test-result-file/

**Reply by wingyplus** [reply]
Thank you very much. 🙇‍♂️
